src/result.js

```diff
@@ -127,29 +127,25 @@ function missingTag(tag) {
 
 function insertIntoIndexHTML(html, head, body) {
   if (!html) { return Promise.resolve(html); }
+  let isBodyReplaced = false;
+  let isHeadReplaced = false;
 
-  if (body) {
-    let isBodyReplaced = false;
-    html = html.replace("<!-- EMBER_CLI_FASTBOOT_BODY -->", function() {
+  html = html.replace(/<\!-- EMBER_CLI_FASTBOOT_(HEAD|BODY) -->/g, function(match, p1) {
```

Comment:
motivation is was improve performance :P,  one `.replace` is better than `.replace`s, simple benchmark I used https://gist.github.com/bekzod/f08628db3cbbbd678bbc8f72e4b01001 , readability-wise I would say it is same level as before at least it uses less lines of code